docs/sources/tempo/metrics-generator/service-graph-view.md

```diff
@@ -1,51 +1,51 @@
 ---
-title: Application performance management
-menuTitle: APM dashboard
+title: Service graph view
+menuTitle: Service graph view
+aliases:
+- /docs/tempo/latest/metrics-generator/app-performance-mgmt
 weight: 200
 ---
 
-# Application performance management dashboard
+# Service graph view
 
-Application performance management (APM) uses data gathered from services, agents, systems, and microservices to visualize and identify areas of concern.
+The service graph view visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
+Once the requirements are set up, this pre-configured view is immediately available.
 
-The Grafana APM dashboard visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
-Once the requirements are set up, this pre-configured dashboard is immediately available.
-
-You can use the APM dashboard to:
+Using the service graph view, you can:
 
 * Discover spans which are consistently erroring and the rates at which they occur
 * Get an overview of the overall rate of span calls throughout your services
 * Determine how long the slowest queries in your service take to complete
 * Examine all traces that contain spans of particular interest based on rate, error and duration values (RED signals)
 
-<p align="center"><img src="../../getting-started/assets/apm-overview.png" alt="APM dashboard"></p>
+<p align="center"><img src="../../getting-started/assets/apm-overview.png" alt="Service graph view"></p>
 
-## Requirements to enable the APM dashboard
+## Requirements
 
 You have to enable span metrics and service graph generation on the Grafana backend so metrics that are generated as traces are ingested.
 
-To use the APM dashboard, you need:
+To use the service graph view, you need:
 
 * Tempo or Grafana Cloud Traces with either 1) the metrics generator enabled and configured or 2) the Grafana Agent enabled and configured to send data to a Prometheus-compatible metrics store
 * [Services graphs]({{< relref "../metrics-generator/service_graphs/#how-to-run" >}}), which are enabled by default in Grafana
-* [APM table enabled](https://grafana.com/docs/grafana/latest/datasources/tempo/) within the service graph tab
+* [Span metrics table enabled](https://grafana.com/docs/grafana/latest/datasources/tempo/) within the service graph tab
```

Comment:
Let's leave this for a separate PR, but I think this is going away now that we've eliminated the feature toggle

Reply:
Actaully, I think that would come out entirely since APM table will be enabled by default, right?